gateway/src/main/java/com/lyft/data/gateway/handler/QueryIdCachingProxyHandler.java

```diff
@@ -38,7 +38,7 @@
   public static final String PROXY_TARGET_HEADER = "proxytarget";
   public static final String V1_STATEMENT_PATH = "/v1/statement";
   public static final String V1_QUERY_PATH = "/v1/query";
-  public static final String QUERY_HTML_PATH = "/query.html";
+  public static final String QUERY_HTML_PATH = "/ui/query.html";
```

Comment:
What's this one fixing, btw?

Reply:
recently noticed that `/query.html` has moved to `/ui/query.html`

Reply:
Ahh, gotcha. Ty!